src/Core/Form/IdentifiableObject/DataFormatter/AbstractFormDataFormatter.php

```diff
@@ -28,11 +28,23 @@
 
 namespace PrestaShop\PrestaShop\Core\Form\IdentifiableObject\DataFormatter;
 
+use PrestaShop\PrestaShop\Core\Util\String\ModifyAllShopsUtil;
 use Symfony\Component\PropertyAccess\Exception\AccessException;
 use Symfony\Component\PropertyAccess\PropertyAccess;
 
-class AbstractFormDataFormatter
+abstract class AbstractFormDataFormatter
 {
+    /**
+     * @var string
+     */
+    protected $modifyAllNamePrefix;
+
+    public function __construct(
+        string $modifyAllNamePrefix
```

Comment:
```suggestion
        string $modifyAllNamePrefix = ''
```

This should be optionable not all formatters will need multishop support so the abstract class shouldn't suggest it's mandatory with this kind of constructor